roles/edpm_container_quadlet/tasks/cleanup.yml

```diff
@@ -1,13 +1,19 @@
 ---
-- name: Stop and disable old standalone container for {{ edpm_container_quadlet_service }}
-  ansible.builtin.systemd_service:
-    name: "edpm_{{ edpm_container_quadlet_service }}"
-    state: stopped
-    enabled: false
-  become: true
 
-- name: Cleanup standalone service file for {{ edpm_container_quadlet_service }}
-  ansible.builtin.file:
-    path: "/etc/systemd/system/{{ edpm_container_quadlet_service }}.service"
-    state: absent
-  become: true
+- name: Cleanup old edpm and tripleo services
+  loop:
+    - "edpm"
+    - "tripleo"
```

Comment:
if this is for adoption, aren't the tripleo services handled already by edpm_tripleo_cleanup/tasks/main.yml?

Reply:
Yeah, it's an attempt to resolve the issue I have with the OVN quadlets PR:
https://github.com/openstack-k8s-operators/edpm-ansible/pull/948#issuecomment-2861476157

But I agree, it would be better to keep everything in one place. It will become cumbersome if we need to handle it separately.